airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -98,6 +99,12 @@
 KUBE_CONFIG_ENV_VAR = "KUBECONFIG"
 
 
+def check_exception_is_kubernetes_api_unauthorized(exc: BaseException):
+    return (
+        isinstance(exc, kubernetes.client.exceptions.ApiException) and exc.status and str(exc.status) == "401"
+    )
```

Comment:
looks good, can u just move this to [utils](https://github.com/apache/airflow/blob/main/airflow/providers/cncf/kubernetes/utils/pod_manager.py) please?

Reply:
yes done 👍

Reply:
a nitpicking can you add please a return type too?